src/cljs/rems/administration/create_form.cljs

```diff
@@ -169,24 +169,25 @@
 (rf/reg-event-fx
  ::send-form
  (fn [{:keys [db]} [_]]
-   (let [edit? (db ::edit-form?)
-         form-errors (validate-form (db ::form) (db :languages))
-         send-verb (if edit? put! post!)]
-     (when-not form-errors
-       (status-modal/common-pending-handler! (page-title edit?))
-       (send-verb (str "/api/forms/" (if edit?
+   (let [edit? (::edit-form? db)
+         form-errors (validate-form (::form db) (:languages db))
+         send-verb (if edit? put! post!)
+         send-url (str "/api/forms/" (if edit?
                                        "edit"
                                        "create"))
-                  {:params (merge (build-request (db ::form) (db :languages))
-                                  (when edit?
-                                    {:form/id (db ::form-id)}))
-                   :handler
-                   (partial status-modal/common-success-handler!
-                            #(dispatch! (str "#/administration/forms/"
-                                             (if edit?
-                                               (db ::form-id)
-                                               (:id %)))))
-                   :error-handler status-modal/common-error-handler!}))
+         description (page-title edit?)
+         request (merge (build-request (db ::form) (db :languages))
+                        (when edit?
+                          {:form/id (db ::form-id)}))]
+     (when-not form-errors
+       (send-verb send-url
+                  {:params request
+                   :handler (flash-message/default-success-handler
+                             description
+                             (fn [response]
+                               (dispatch! (str "#/administration/forms/" (or (::form-id db)
```

Comment:
This seems to have changed back to how it had been previously before a recent fix: compare this line to L186-188 in the old version.

Reply:
hmm maybe I botched the merge

Reply:
fixed now